src/components/Slider/Slider.js

```diff
@@ -22,6 +22,11 @@ class Slider extends React.Component {
   }
 
   render() {
+    // There's no need to try to render the Slider, it will not be usable
+    if (this.props.range.min === this.props.range.max) {
```

Comment:
Forget about it, if it goes to empty -> with content, we'll end up recreating some DOM nodes.